uvm/servlets/admin/app/model/Session.js

```diff
@@ -28,14 +28,14 @@ Ext.define ('Ung.model.Session', {
         { name: 'remoteAddr', type: 'auto' },
         { name: 'serverCountry', type: 'string' },
         { name: 'serverIntf', type: 'auto' },
-        { name: 'serverKBps', type: 'number', convert: function (val) { return val ? Math.round(val*1000)/1000 : null; } },
+        { name: 'serverBps', type: 'number', convert: function (val) { return val ? Math.round(val*1000)/1000 : null; } },
```

Comment:
Shouldn't this just be returning the value as-is now?

Reply:
removed that part as it's not needed now